server/app/views/questiontypes/CurrencyQuestionFragment.html

```diff
@@ -2,8 +2,13 @@
   th:fragment="currency(question, questionRendererParams)"
   th:with="currencyQuestion=${question.createCurrencyQuestion()},
              questionId=${'id-' + #strings.randomAlphanumeric(8)},
-             inputId=${'id-' + #strings.randomAlphanumeric(8)}"
+             inputId=${'id-' + #strings.randomAlphanumeric(8)},
+             fieldErrors=${currencyQuestion.getValidationErrors().get(currencyQuestion.getCurrencyPath())},
```

Comment:
question (non-blocking):  Do we need to differentiate field errors and question errors since currency questions only have one field?

Reply:
In currency questions I think I can take it out, but there are questions that have their own field errors even when there is only one input because there is additional validation (text questions with max length etc). I'll go through and make sure I'm only including fieldErrors for question types where they exist